README.md

```diff
@@ -1,70 +1,226 @@
+In order to run this project you need:
 
-### `npm run build` fails to minify
+-visual studio
+-github
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+### Setup
+
+Clone this repository to your desired folder:
+
+```
+git clone https:https://github.com/edahigure/space-travelers-hub.git
+I run this command
+npx create-react-app my-app  to install react, react-dom, and react-scripts with cra-template...
+```
+
+### Install
+
+- Install all the dependencies required for the website. By using the command below.
+```
+Run npm install to install all dev dependencies for Linters, Webpack, Jest & Babel.
+```
+  
+- Open the index.html file in a browser.
+-
+
+### Install
+
+Install this project with:
+
+```
+  npm install
+```
+### Usage I
+
+Change the main branch to the right one, execute the following command:
+
+```sh
+  git checkout event
+```
+
+### Usage II
+
+To run the project, execute the following command:
+
+```sh
+  npm start
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+To run the webhint linter
+
+```sh
+  npx hint .
+```
+
+To run the stylehint linter
+
+```sh
+  npx stylehint "**/*.{css,scss}"
+```
+
+To run the ESLint linter
+
+```sh
+  npx eslint "**/*.{js,jsx}" 
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHOR -->
+
+## 👥 Author <a name="author"></a>
+
+👤 **TUMBU BARTHOLOMEW ELAD**
+GitHub: [@githubhandle](https://github.com/elad237)
+- Twitter: [@twitterhandle](https://twitter.com/Elad59380989)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/tumbu-elad-896ab2183/)
+
+👤 **Edahi Antonio Gutiérrez Reyes**
+GitHub: [@githubhandle](https://github.com/edahigure)
+- Twitter: [@twitterhandle](https://twitter.com/edahigure)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/edahigure/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- **Tests**
+- **Deploy**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/edahigure/space-travelers-hub/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project follow me and star it!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank Microverse<br>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Kindly fix the MIT license link as requested the previous reviewer. Currently the link is blanc